static/sw.js

```diff
@@ -0,0 +1,10 @@
+importScripts('/bower_components/sw-toolbox/sw-toolbox.js');
+
+toolbox.options.debug = true;
+toolbox.router.default = toolbox.fastest;
```

Comment:
Based on the staging site, there are a number of requests repeated on each pageload that could be added to the cache using either cacheFirst or cacheOnly:

* favicon.ico
* GitHub-Mark-Light-64px.png
* favicon-*.png

It may also make sense to have the cache header be something other than maxage=1 for these